core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -73,7 +74,8 @@ class WorkflowSubmissionActor(val dataSource: SlickDataSource,
                               override val workbenchMetricBaseName: String,
                               val requesterPaysRole: String,
                               val useWorkflowCollectionField: Boolean,
-                              val useWorkflowCollectionLabel: Boolean) extends Actor with WorkflowSubmission with LazyLogging {
+                              val useWorkflowCollectionLabel: Boolean,
+                              val defaultBackend: String) extends Actor with WorkflowSubmission with LazyLogging {
```

Comment:
Not directly related to your PR. It'd be nice at some point we can have a `WorkflowSubmissionActorConfig` case class so that we don't have to modify the constructor every time.